modules/recommended-requirements-central-services.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-recommended-requirements.adoc
+:_content-type: CONCEPT
+[id="recommended-requirements-central-services_{context}"]
+= Central services (self-managed)
+
+Central services contain the following components:
+
+* Central
+* Scanner
+
+[NOTE]
+====
+If you are using ACS:CS, you do not need to look at requirements of central services, because they are managed by RedHat. You can look further down to look for requirements of Secured cluster services.
```

Comment:
```suggestion
If you are using {rh-rhacscs-first}, you do not need to review the requirements for Central services, because they are managed by Red Hat. You only need to look at the requirements for secured cluster services.
```